ci-operator/step-registry/baremetal/lab/upi/conf/grub2/baremetal-lab-upi-conf-grub2-commands.sh

```diff
@@ -55,8 +55,8 @@ load_video
 menuentry 'Install ($flavor)' {
     set gfx_payload=keep
     insmod gzio
-    linux  /${CLUSTER_NAME}/vmlinuz debug nosplash ip=${baremetal_iface}:dhcp $kargs coreos.live.rootfs_url=http://${INTERNAL_NET_IP}/${CLUSTER_NAME}/rootfs.img ignition.config.url=http://${INTERNAL_NET_IP}/${CLUSTER_NAME}/$mac_postfix-console-hook.ign ignition.firstboot ignition.platform.id=metal
-    initrd /${CLUSTER_NAME}/initramfs.img
+    linux  /${CLUSTER_NAME}/vmlinuz_${arch} debug nosplash ip=${baremetal_iface}:dhcp $kargs coreos.live.rootfs_url=http://${INTERNAL_NET_IP}/${CLUSTER_NAME}/rootfs-${arch}.img ignition.config.url=http://${INTERNAL_NET_IP}/${CLUSTER_NAME}/$mac_postfix-console-hook.ign ignition.firstboot ignition.platform.id=metal
```

Comment:
@aleskandro are we re-using the ignition file from the installation to add the day 2 workers? We may want to consider getting the worker.ign from the cluster instead: https://access.redhat.com/solutions/5504291
Just to cover in case day 2 isn't literally day 2?

Reply:
see offline discussion